tests/data/phpDocumentor2ExpectedArray.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
```

Comment:
Licence and docblocks are missing here. Not all methods need a docblock. But please explain why this class was needed.